pkg/bosh/bpmconverter/volume_factory.go

```diff
@@ -358,29 +444,18 @@ func resolvedPropertiesVolumeMount(name string, instanceGroupName string) corev1
 	}
 }
 
-// For ephemeral job data.
-// https://bosh.io/docs/vm-config/#jobs-and-packages
-func dataDirVolume() *corev1.Volume {
-	return &corev1.Volume{
-		Name:         VolumeDataDirName,
-		VolumeSource: corev1.VolumeSource{EmptyDir: &corev1.EmptyDirVolumeSource{}},
-	}
-}
-
-func dataDirVolumeMount() *corev1.VolumeMount {
-	return &corev1.VolumeMount{
-		Name:      VolumeDataDirName,
-		MountPath: VolumeDataDirMountPath,
-	}
-}
-
 func sysDirVolume() *corev1.Volume {
 	return &corev1.Volume{
 		Name:         VolumeSysDirName,
 		VolumeSource: corev1.VolumeSource{EmptyDir: &corev1.EmptyDirVolumeSource{}},
 	}
 }
 
+// VolumeDataDirName is the volume name for the data directory.
```

Comment:
Does this have to be public? I can't find any usage  outside `bpmconverter`.
In the tests I would prefer to use the constant string result. Feels more decoupled from the implementation. Also  eliminates the need for `VolumeDataDirName` unit tests.

Actually, if this is only used in bpmconverter, the func can be replaced with `generateEphemeralPersistentVolumeClaimName` completely.